datajunction-clients/python/datajunction/_base.py

```diff
@@ -0,0 +1,59 @@
+"""Base client dataclasses."""
+from dataclasses import fields, is_dataclass
+from typing import TYPE_CHECKING, Any, Dict, Type, TypeVar, Union, get_args, get_origin
+
+if TYPE_CHECKING:  # pragma: no cover
+    from datajunction.client import DJClient
+
+
+T = TypeVar("T")
+
+
+class SerializableMixin:  # pylint: disable=too-few-public-methods
```

Comment:
This could be part of _internal module, but no big deal either way.

Reply:
Yeah, I originally had it there but it felt like that file got really long. This piece is pretty generic whereas that module is now mostly client models.